pki.spec

```diff
@@ -49,7 +49,11 @@ Source: https://github.com/dogtagpki/pki/archive/v%{version}%{?_phase}/pki-%{ver
 # Java
 ################################################################################
 
+%if 0%{?fedora} && 0%{?fedora} >= 33
+%define java_home %{_usr}/lib/jvm/java-11-openjdk-11.0.7.10-1.fc32.x86_64
```

Comment:
Is there a more generic path, e.g. `%{_usr}/lib/jvm/java-11-openjdk-11`?

Reply:
The `java-11-openjdk-headless` package provides the `/usr/lib/jvm/jre-11-openjdk-11.0.7.10-1.fc32.x86_64` path. No package actually owns `/usr/lib/jvm/java-11-openjdk`, but it is provided by the `alternatives` system. This means it can't be used for dependency management... We could still use it though, I suppose. Thoughts? (The existing path is provided by a package specifically.)

Reply:
So IIUC I think for Fedora 33+ we can use `java-11-openjdk-headless` as dependency and `/usr/lib/jvm/java-11-openjdk` (which we assume will point to the correct folder) as JAVA_HOME. I was just concerned if the path is too specific it will break as soon as they update OpenJDK.

Reply:
OK. Can we do that for `java-8-openjdk` as well?

Reply:
I think so. That'll make it consistent.

Reply:
Done.